community/cypher/cypher/src/main/java/org/neo4j/cypher/internal/javacompat/ExecutionEngine.java

```diff
@@ -54,12 +54,12 @@ public ExecutionEngine( GraphDatabaseQueryService queryService, LogProvider logP
     }
 
     @Override
-    public Result executeQuery( String query, MapValue parameters, TransactionalContext context )
+    public Result executeQuery( String query, MapValue parameters, TransactionalContext context, Boolean prePopulate )
```

Comment:
I think it is better to use primitive `boolean` in java. Otherwise `null` can be given as an argument.